base-depends

```diff
@@ -13,6 +13,13 @@ gstreamer1.0-plugins-good
 gstreamer1.0-tools
 gstreamer1.0-vaapi
 ibus-gtk3
+# PipeWire must be listed before libcanberra-pulse, otherwise
+# it'll pull PulseAudio
+libspa-0.2-bluetooth
+pipewire
+pipewire-audio-client-libraries
+pipewire-pulse
+wireplumber
```

Comment:
I think just `pipewire-pulse` needs to move here. Apt can still order the installation correctly if all the other ones come later. All we're trying to do is help it figure out that we prefer `pipewire-pulse` to `pulseaudio` at the time it encounters the optional dependency in `libcanberra-pulse`. I think it's better to leave the other ones where they were to maintain the otherwise alphabetical ordering.

Reply:
Sure